engines/python/src/main/java/ai/djl/python/engine/PyProcess.java

```diff
@@ -124,6 +124,7 @@ synchronized void startPythonProcess() {
             int id = restartCount.get();
             int port = connections.get(0).getPort();
             logger.info("Start process: {} - retry: {}", port, id);
+            pyEnv.createVirtualEnv(model.getName());
```

Comment:
Using venv should be disabled by default.

Reply:
Done.